Task 1.2

rafazgr opened the pull request.

rafazgr committed:
Task 1.2

IgorKheidorov commented on Task1.2/Program.cs:
```diff
@@ -0,0 +1,38 @@
+        Console.WriteLine("Enter the first 9 digits of the ISBN:");
+        string id = Console.ReadLine();
+
+        int checkDigit = CalculateCheckDigit(id);
```
use ternary  isbn += checkDiFigit == 10 ? "X" : checkDigit.ToString();

IgorKheidorov reviewed (approved):
Remove binaries from git

rafazgr committed:
Update Program.cs

Modified to:

use ternary isbn += checkDiFigit == 10 ? "X" : checkDigit.ToString();
